css/DayPicker.scss

```diff
@@ -57,6 +57,56 @@
   left: 50%;
 }
 
+.DayPicker.DayPicker--scrollable {
+  height: 100%;
+
+  .DayPicker__week-header {
+    top: 0;
+    display: table-row;
+    border-bottom: 1px solid $react-dates-color-border;
+    background: white;
+  }
+
+  .CalendarMonth__caption {
```

Comment:
So ideally, we don't want any `CalendarMonth` stylings to be in the `DayPicker.scss` file. Having that confusion is going to make converting to `react-with-styles` more of a pain. The solution is to pass down the scrollable prop as far as it needs to go down the tree and add a `CalendarMonth--scrollable` class and a `DayPickerNavigation--scrollable` class and so forth.